### ISSUE TITLE
Clear old entries of the queue

### ISSUE THREAD

**Issue by liquidnya** [fluid-queue/fluid-queue#5]
Add a script to clear old queue entries and also the waiting time and the last online time of an user.
For example: remove all entries, where someone did not show up to the stream in the last month.

To add that script, `cleanup.js` could be created and also new settings  could be created to configure when a user should be removed from the queue.
When the `cleanup.js` script is used, one could add it to the `package.json` scripts https://github.com/fluid-queue/fluid-queue/blob/ada8a9da95deef5eda37b658ef67f64841201b9a/package.json#L22
Alternatively this could also be implemented by adding a broadcaster only `!cleanup` command or similar.

Having the queue make a backup of the queue, waiting times and online user times before cleanup the queue would also be a nice bonus.

_Originally posted by @liquidnya in https://github.com/ToransuShoujo/quesoqueue_plus/issues/11#issue-1246082668_

**Comment by demize** [comment]
@liquidnya I think you've mostly done this already, through an alternate approach of modifying the `!clear` command :)

**Comment by liquidnya** [comment]
@demize yes, this will be fixed by #101. Tests are incomplete still and it is currently missing a rebase too.

**Comment by liquidnya** [comment]
I decided against a `npm run cleanup` command because of the usage of docker.
I also decided to re-use the `!clear` command instead of introducing a new `!cleanup` command, because I am worried that people might mix up the `!clear` and `!cleanup` commands accidentally clearing all levels. Also with the new solution accidentally using `!clear NB0-1MD-SLG` (This is used by maker teams to mark a level cleared in the team) will not clear all levels. The PR also has an option to make using `!clear` the default to clear all levels again (by setting the new `clear` setting to `"all"`).
